### PR TITLE
Create Dockerfiles to build olm files for openshift and Kubernetes

### PR DESCRIPTION
**Pull request by flacatus**
Created docker images of a dedicated catalog

Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### CODE REVIEW

**Comment by rhopp** [comment]
[terst]

**Comment by rhopp** [comment]
[test]

**Comment by tolusha** [comment]
ci-build

**Comment by tolusha** [comment]
[test]

**Comment by tolusha** [comment]
[build]

**Comment by tolusha** [comment]
[test]

**Comment by tolusha** [comment]
@rhopp 
How to trigger build ?

File: olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1567522875/eclipse-che-preview-kubernetes.v9.9.9-nightly.1567522875.clusterserviceversion.yaml [hunk 1]
```diff
@@ -350,5 +350,5 @@ spec:
   maturity: stable
   provider:
     name: Eclipse Foundation
-  replaces: eclipse-che-preview-kubernetes.v9.9.9-nightly.1567514659
+  replaces: eclipse-che-preview-kubernetes.v9.9.9-nightly.1567437268
```
Comment (tolusha) [inline olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1567522875/eclipse-che-preview-kubernetes.v9.9.9-nightly.1567522875.clusterserviceversion.yaml]: is @davidfestal fine with these changes?

**Comment by AndrienkoAleksandr** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by davidfestal** [comment]
@tolusha I approved with a comment (to answer your question here: https://github.com/eclipse/che-operator/pull/165#discussion_r375675066)

**Comment by tolusha** [comment]
@flacatus 
It is ok to merge?

**Comment by flacatus** [comment]
@tolusha For me is prepared to be merged. Anyway I need another PR to prepare a script which build the dockers locally and run them...What you think?

**Comment by tolusha** [comment]
@flacatus It  can be another PR

**Pull request merged**